apps/homescreen/js/app.js

```diff
@@ -540,6 +577,12 @@ const SETTINGS_VERSION = 0;
 
         this.pageHeight = pageHeight;
         this.pendingGridHeight = gridHeight;
+
+        if (!this.visualLoadComplete &&
+            Math.floor(visibleChildren / (this.small ? 4 : 3)) * iconHeight >=
+              scrollHeight) {
+          this.onVisualLoad();
+        }
```

Comment:
Ok, now I do see it and it's quite obvious - I'm going to take a look and see what it is.